0000-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,104 @@
+# HIP XX: <--insert title
+
+- Author: @HeatedLime
+- Start Date: 5/12/2023
+- Category: Technical & Economic
+- Original HIP PR: #652
+- Tracking Issue: 
+- Voting Requirements: veIOT
+
+## Summary:
+This Helium Improvement Proposal (HIP) suggests adding a hex multiplier score to the MOBILE Proof of Coverage (PoC) modeled coverage points based on whether other coverage from Helium 5g deployments exist within that res12 hex. 
+
+## Motivation:
+HIP 74 was passed to incorporate obstruction data and radio signal power into the PoC reward model; however, it weighed all coverage within each res12 hex equally, even if multiple radios were already providing coverage within that res12 hex. This means deployers could point 5 outdoor radios in the same direction, and still be awarded full modeled coverage points for each res12 hex. 
+
+This proposal aims to improve the value of the network coverage by incentivizing users to deploy radios that minimize overlapping coverage, and encourage deployments in new areas. 
+
+## Stakeholders:
+The stakeholders of this proposal are radio deployers. 
+ 
+## Detailed Explanation:
+Currently, any redundant network coverage is still rewarded with the same. This discourages the buildout of coverage to new areas. To prevent overcrowding and overlapping of coverage in hexes, this HIP proposes to limit the amount of modeled coverage points radios are awarded to redundant coverage in res12 hexes. 
+
+To ensure that only the best setups are rewarded, only the top four (4) radio signals in each res12 hex will be awarded modeled coverage points, with a decaying multiplier based on the radio score noted below. Any radios not scored within the top four (4) will be graded as “Fail”. 
+
+
+Score/Grade
+Multiplier 
+1
+1X
+2
+.75X
+3
+.5X
+4
+.25X
+Fail
+0X
+
+
+All outdoor radios that provide coverage to any res12 hex will be given a score for each res12 hex they provide coverage in based on the following potential attributes (note, this score is only for a single res12 hex and not the entire radio):
+
+### Modeled Signal Strength 
+Date of CPI approval (only used as a tiebreaker if tie for attribute 1)
```

Comment:
I would bullet point these.

List all the following potential attributes in bullet points.

Reply:
Added bullets for both.